docs/UserGuide.md

```diff
@@ -893,11 +954,22 @@ delivery? No worries! Just edit the details of a delivery simply by using this f
 I see that you've just shipped out an order. Nice! Now you can update the status of that particular delivery
 to SHIPPED! Using this feature, you can update the status of a delivery accordingly.
 
+
 <box background-color="#d9edf7" border-color="#bce8f1" type="tip" header="Tip">
     Delivery Status typed in need not be in uppercase.
 </box>
 
-**Format:** `delivery status CUSTOMER_ID STATUS`
+**Format:** `delivery status DELIVERY_ID STATUS`
+
+
+<box type="note" background-color="#dff0d8" border-color="#d6e9c6" icon=":information_source:">
+
+**Note:**
+
+* `DELIVERY_ID`{.swfit} must be an integer greater than 0.
```

Comment:
`DELIVERY_ID`{.swift} must be an integer greater than 0 that corresponds to an existing Delivery in the address book.